src/os.c

```diff
@@ -86,22 +86,21 @@ char os_secure_memcmp(const void *src1, const void *src2, size_t length)
 
 #ifndef HAVE_BOLOS
 #define MAIN_LINKER_SCRIPT_LOCATION 0xC0DE0000
+#include "lib_standard_app/debug.h"
```

Comment:
Why do you need this here?

Reply:
I don't think this will work on all apps.
they will have the .h, but not the .c if not using standard_app, hence they can't use `HAVE_DEBUG_THROWS`, which is not really explicit?